### ISSUE TITLE
General discussions and supporting information

### ISSUE THREAD

**Issue by DeAngelisA** [UniLuFS2020-ReplicationSeminar/test-replication#4]
I keep this issue open to any question or comment or help requests.

**Comment by DeAngelisA** [comment]
# Adding files to .gitignore

This is a general point, as it can touch many of the replications: if you have sensitive data, you should not share it. For instance, I have seen data files in @ato-h-96 and @DanSchlaeger folder, check that out. 
For more information refer to the section [16.4 Stage and Commit](https://happygitwithr.com/existing-github-first.html) of the Happy Git for the UseR book.
See in particular this:
![image](https://user-images.githubusercontent.com/9892396/78809098-ae1f6500-79c6-11ea-9493-1de96694ca34.png)

Basically, an easy way to take care of this is placing all the files in a e.g. /data folder, and then add "/data" in the `.gitignore` file (you can do that with any text processor, like block note).

**Comment by DeAngelisA** [comment]
Check out this tweet showing various ggplot geomtries: [https://twitter.com/andrewheiss/status/1247322469989572610?s=21](https://twitter.com/andrewheiss/status/1247322469989572610?s=21).
Code is in a reply link!
